boggle_blast.lic

```diff
@@ -0,0 +1,165 @@
+      while line = get
+        if line =~ /^The pumpkin farmer (beams happily|grins) at/
+          response = bput('give farmer 200 dokoras', 'The farmer takes your money', 'Sorry, the game rooms are all full right now')
+          next if response =~ /Sorry, the game rooms are all full right now/
+          break if response =~ /The farmer takes your money/
+        end
+      end
+    end
+    5.times do
+      @true_pumpkin = false
+      get_pumpkin while !@true_pumpkin
+      case bput('load slingshot', 'You carefully load', "You need to have a pumpkin")
+      when "You need to have a pumpkin"
+        stow_hands
+        @true_pumpkin = false
+        get_pumpkin while !@true_pumpkin
+        bput('load slingshot', 'You carefully load')
+      end
+      case bput('fire slingshot', 'You position the slingshot', 'You let loose', 'You carefully aim the', 'You really should be standing to play', 'You fire the slingshot and awkwardly', 'You take careful aim')
+      when 'You really should be standing to play'
+        bput('stand', 'You stand back up')
+        bput('fire slingshot', 'You position the slingshot', 'You let loose', 'You carefully aim the', 'You fire the slingshot and awkwardly', 'You take careful aim')
+      end
+      pause
+    end
+    pause
+    case bput("go door", "Obvious exits", "You must be standing to do that")
+    when "You must be standing to do that"
+      bput('stand', 'You stand back up')
+      bput("go door", "Obvious exits")
+    end
+    bput("get prize", "The pumpkin farmer beams happily at you", "The pumpkin farmer grins at you")
+    pause
+  end
+
+  def get_pumpkin
+    @true_pumpkin = false
+    case bput('get pumpkin', 'You help yourself', 'You pick up')
+    when 'You pick up'
+      if @fake_pumpkins.any? { |item| Regexp.new(item) =~ right_hand }
+        stow_hands
+      else
+        @true_pumpkin = true
+      end
+    else
+      @true_pumpkin = true
+    end
+  end
+
+  def check_prize
+    if @trash_items.any? { |item| Regexp.new(item) =~ right_hand }
+      bput("put my #{righthand} in bucket", "You put", "You drop")
+    elsif @danger_items.any? { |item| Regexp.new(item) =~ right_hand }
+      pause
+      bput("put my doll in bucket", "An odd sort of noise comes from your doll")
+    elsif right_hand.include?('wallet')
+      bput("put my wallet in my haversack", "You put")
+    elsif right_hand.include?('fan')
+      bput("close my fan", "With a practiced")
+      bput("put my #{right_hand} in bucket", "You put", "You drop")
+    elsif right_hand.include?('ticket')
+      bput("get ticket in my #@stow_container", "You get")
+      case bput("combine my ticket", "You combine", "You can't have")
+      when "You can't have"
+        bput("stow right", "You put")
+        pause
+        bput("put my ticket in my #@full_container", "You put")
```

Comment:
❌ More string interpolation: `#{@full_container}`

Reply:
Downgraded to 💡

Reply:
This uses compact string interpolation and works.  There's no real reason not to change it to standard interpolation if you'd prefer it conform to a style preference.